config/samples/manila_v1beta1_manilaapi.yaml

```diff
@@ -4,3 +4,10 @@ metadata:
   name: manilaapi-sample
 spec:
   # TODO(user): Add fields here
+  #tls:
+  #    api:
+  #      internal:
+  #        secretName: cert-manila-internal-svc
+  #      public:
+  #        secretName: cert-manila-public-svc
+  #    caBundleSecretName: combined-ca-bundle
```

Comment:
as we have a dedicated sample, I think we could drop this here, but its really just cosmetic

Reply:
Agree, it's a remnant from previous config. Ty!